source/id-ID/1.0.0/index.html.haml

```diff
@@ -0,0 +1,327 @@
+---
+description: Pencatatan Changelog
+title: Pencatatan Changelog
+language: id-ID
+version: 1.0.0
+---
+
+- changelog = "https://github.com/olivierlacan/keep-a-changelog/blob/master/CHANGELOG.md"
+- gemnasium = "https://gemnasium.com/"
+- gh = "https://github.com/olivierlacan/keep-a-changelog"
+- issues = "https://github.com/olivierlacan/keep-a-changelog/issues"
+- semver = "http://semver.org/"
+- shields = "http://shields.io/"
+- thechangelog = "http://5by5.tv/changelog/127"
+- vandamme = "https://github.com/tech-angels/vandamme/"
+- iso = "http://www.iso.org/iso/home/standards/iso8601.htm"
+- ghr = "https://help.github.com/articles/creating-releases/"
+- aditya = "https://github.com/adityapurwa/"
+
+.header
+  .title
+    %h1 Pencatatan Changelog
+    %h2 Standarisasi pencatatan Changelog untuk kolaborasi yang lebih baik
+
+  = link_to changelog do
+    Versi
+    %strong= current_page.metadata[:page][:version]
+
+  %pre.changelog= File.read("CHANGELOG.md")
+
+.answers
+  %h3#what
+    %a.anchor{ href: "#what", aria_hidden: "true" }
+    Apa itu changelog?
+
+  %p    
+    Changelog adalah sebuah file yang berisi daftar perubahan yang
+    diurutkan secara kronologis untuk setiap versi dari sebuah project.
+
+  %h3#why
+    %a.anchor{ href: "#why", aria_hidden: "true" }    
+    Kenapa perlu untuk mencatat changelog?
+
+  %p    
+    Supaya pengguna dan kontributor lebih mudah melihat perubahan
+    apa saja yang terjadi di setiap rilis (atau versi) dari sebuah project.
+
+  %h3#who
+    %a.anchor{ href: "#who", aria_hidden: "true" }
+    Siapa yang butuh changelog?
+
+  %p    
+    Semua orang membutuhkannya. Baik pengguna ataupun pengembang, setiap
+    orang yang menggunakan perangkat lunak adalah manusia yang peduli dengan
+    apa yang ada di perangkat lunak tersebut. Mereka ingin tahu apa 
+    dan kenapa terjadi perubahan.
+
+.good-practices
+  %h3#how
+    %a.anchor{ href: "#how", aria_hidden: "true" }
+    Bagaimana cara membuat changelog yang baik?
+
+  %h4#principles
+    %a.anchor{ href: "#principles", aria_hidden: "true" }
+    Prinsip Dasar
```

Comment:
What about the `Prinsip - Prinsip Dasar`

Reply:
I thinks that's better as it is plural, will update.